# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/notification/PNotification.ipdl
# Commit: cc25e5c94813
# Full Hash: cc25e5c94813aac3d7043350f3d1e090e30d2363
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2024-11-12 09:32:50
# Regressor Bug: 1928702
# File Overlap Count: 1
# Description:
#   Bug 1928702 - Part 1: Add skeleton PNotification r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D227686
# ==============================================================================

diff -r af9b668bb659 -r cc25e5c94813 dom/notification/PNotification.ipdl
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/dom/notification/PNotification.ipdl	Mon Nov 11 20:34:11 2024 +0000
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+include protocol PBackground;
+
+include "ipc/ErrorIPCUtils.h";
+
+using mozilla::CopyableErrorResult from "mozilla/ErrorResult.h";
+
+namespace mozilla {
+namespace dom {
+namespace notification {
+
+[ChildProc=anydom]
+async protocol PNotification {
+ parent:
+  async Show() returns (CopyableErrorResult rv);
+};
+
+} // namespace notification
+} // namespace dom
+} // namespace mozilla